access-control-app/docs/api.md

```diff
@@ -863,6 +864,20 @@ Content-Type: application/json;charset=ISO-8859-1
 {"C1200000000-PROV1": ["read"], "C1200000001-PROV1": []}
 ```
 
+When requesting permissions for large sets of concept ids, you can make a POST request to `%CMR-ENDPOINT%/permissions` and pass the query parameters in the POST body.
```

Comment:
change `query parameters` to `query parameters in URL encoded query string`.

Reply:
Fixed